autoload/search.vim

```diff
@@ -325,9 +327,27 @@ endfu
 fu! s:RunRgDefinitionSearch(language, patterns) abort
   let rg_ft = s:GetRgFiletype(a:language)
 
-  let cmd = s:rg_base_cmd . ' -t ' . rg_ft
+  " TODO: instead of disabling the .gitignore filtering
+  " is there some way to just null out any `scan_results`
+  " patterns which match current `.gitignore` patterns?
+  let cmd = s:rg_base_cmd . ' -u' . ' -t ' . rg_ft
+  " let cmd = s:rg_base_cmd . ' -t ' . rg_ft
   let cmd = cmd . s:GetRgIgnoreSpecifier()
-  let cmd = cmd . ' ' . a:patterns
+
+  " NOTE: the `-e` is ok here to be explicit right?
+  let cmd = cmd . ' -e ' . a:patterns
+
+  let scan_cmd = g:any_jump_glob_scanner
+  if strlen(scan_cmd)
+      let scan_results = system(scan_cmd)
+      " echo 'Additional ' a:language . ' PATHs to scan: ' . scan_results
+      let cmd = cmd  . ' ' . scan_results
+  endif
+
+  " TODO: is it more correct to pass `-g <glob>` flags here?
```

Comment:
Would like feedback on whether this (or something else) is a better approach?

Also see the TODOs in the description.